kaldb/src/main/java/com/slack/kaldb/logstore/LuceneIndexStoreImpl.java

```diff
@@ -139,9 +141,13 @@ private IndexWriterConfig buildIndexWriterConfig(
     final IndexWriterConfig indexWriterCfg =
         new IndexWriterConfig(analyzer)
             .setOpenMode(IndexWriterConfig.OpenMode.CREATE)
-            .setRAMBufferSizeMB(config.ramBufferSizeMB)
-            .setUseCompoundFile(false)
```

Comment:
Using the defaults here is better unless we have an explicit need to override them. The benefit with this is as we upgrade versions we float with the recommended best approaches instead of having old overrides that are forgotten.

I'll split this and rambuffer change into a separate PR though.

Reply:
https://github.com/slackhq/kaldb/pull/313